templates/default.nginx_http.conf.template

```diff
@@ -1,14 +1,94 @@
+# Expires map
+map $sent_http_content_type $expires {
+    default                    off;
+    text/html                  epoch;
+    text/css                   86400;
+    application/javascript     86400;
+    application/octet-stream   604800;
+    ~image/                    604800;
+}
+
 server {
-    listen 80;
-    listen [::]:80;
-    server_name  {{ DOMAIN_NAME }};
+    server_name  default_server;
+    include /etc/nginx/conf.d/default.nginx_http.shared.conf;
+    # listen       4200 http2;
```

Comment:
This seems like a possible issue. HTTP/2 wont work without ssl afaik. We might still SSL the connection to the load balancer if we want to support it. In general I favor supporting HTTP/2 if we can however having two separated SSL boundaries seems odd.

Reply:
Currently the connection from the load balancer to here is over http, not https.